### PR TITLE
[FIX] stock: give correct field name to prepare stock return picking

### PR DESCRIPTION
**Pull request by alsh-odoo**
This traceback arises when the user clicks on the `return` button.

To reproduce this issue:

1) Install `stock`
2) Activate `Multi-Step Routes` from `Inventory/configuration/settings`
 3) Create a new `product` with `quantity` 100.
4) Create a new stock picking record in `Operations/Deliveries`
5) Select the created product in `operations` with `Demand` as 10 
6) Click on the `Validate` button and then the `Return` button 
7) Click on the `Return` button and again click the `return` button for that picking type

Error: 
```
KeyError: 'reserved_qty'
  File "odoo/http.py", line 2157, in __call__
    response = request._serve_db()
  File "odoo/http.py", line 1732, in _serve_db
    return service_model.retrying(self._serve_ir_http, self.env)
  File "odoo/service/model.py", line 133, in retrying
    result = func()
  File "odoo/http.py", line 1759, in _serve_ir_http
    response = self.dispatcher.dispatch(rule.endpoint, args)
  File "odoo/http.py", line 1960, in dispatch
    result = self.request.registry['ir.http']._dispatch(endpoint)
  File "addons/website/models/ir_http.py", line 235, in _dispatch
    response = super()._dispatch(endpoint)
  File "odoo/addons/base/models/ir_http.py", line 207, in _dispatch
    result = endpoint(**request.params)
  File "odoo/http.py", line 722, in route_wrapper
    result = endpoint(self, *args, **params_ok)
  File "addons/web/controllers/dataset.py", line 24, in call_kw
    return self._call_kw(model, method, args, kwargs)
  File "addons/web/controllers/dataset.py", line 20, in _call_kw
    return call_kw(request.env[model], method, args, kwargs)
  File "odoo/api.py", line 466, in call_kw
    result = _call_kw_multi(method, model, args, kwargs)
  File "odoo/api.py", line 453, in _call_kw_multi
    result = method(recs, *args, **kwargs)
  File "addons/web/models/models.py", line 1076, in onchange
    snapshot1 = RecordSnapshot(record, fields_spec)
  File "addons/web/models/models.py", line 1163, in __init__
    self.fetch(name)
  File "addons/web/models/models.py", line 1178, in fetch
    self[field_name] = self.record[field_name]
  File "odoo/models.py", line 6570, in __getitem__
    return self._fields[key].__get__(self, type(self))
  File "odoo/fields.py", line 1150, in __get__
    self.recompute(record)
  File "odoo/fields.py", line 1361, in recompute
    apply_except_missing(self.compute_value, recs)
  File "odoo/fields.py", line 1334, in apply_except_missing
    func(records)
  File "odoo/fields.py", line 1383, in compute_value
    records._compute_field_value(self)
  File "odoo/models.py", line 4848, in _compute_field_value
    fields.determine(field.compute, self)
  File "odoo/fields.py", line 101, in determine
    return needle(*args)
  File "addons/stock/wizard/stock_picking_return.py", line 65, in _compute_moves_locations
    product_return_moves_data.update(wizard._prepare_stock_return_picking_line_vals_from_move(move))
  File "addons/stock/wizard/stock_picking_return.py", line 86, in _prepare_stock_return_picking_line_vals_from_move
    quantity -= sum(move.move_line_ids.mapped('reserved_qty'))
  File "odoo/models.py", line 6059, in mapped
    recs = recs._fields[name].mapped(recs)
```

See: https://github.com/odoo/odoo/blob/c2b160fe50c24e9f01d0d5b350f2422cfdbc72d4/addons/stock/wizard/stock_picking_return.py#L86

Because the `reserved_qty` field is removed from this PR:- https://github.com/odoo/odoo/pull/137864

sentry-4621794958

### CODE REVIEW

**Comment by alsh-odoo** [comment]
Hello @odoo/sentry_reviewers 

Can you review this PR?

Thank You!!

File: addons/stock/wizard/stock_picking_return.py [hunk 1]
```diff
@@ -83,7 +83,7 @@ def _prepare_stock_return_picking_line_vals_from_move(self, stock_move):
             if not move.origin_returned_move_id or move.origin_returned_move_id != stock_move:
                 continue
             if move.state in ('partially_available', 'assigned'):
-                quantity -= sum(move.move_line_ids.mapped('reserved_qty'))
+                quantity -= sum(move.move_line_ids.mapped('quantity_product_uom'))
```
Comment (adwid) [inline addons/stock/wizard/stock_picking_return.py:86]:
```suggestion
                quantity -= sum(move.move_line_ids.mapped('quantity'))
```

**Comment by alsh-odoo** [comment]
Hello @adwid 

Changes are done.

Thank You!!

**Comment by adwid** [comment]
robodoo r+

**Pull request closed without merging**